dev/com.ibm.ws.httpservice/test/com/ibm/ws/httpsvc/test/SessionTest.java

```diff
@@ -154,6 +155,11 @@ public String getHeader(String name) {
             return null;
         }
 
+        @Override
```

Comment:
Not sure how this compiles.  getHeader and getTrustedHeader are not on HttpRequest.